src/en/news/blog/2022/scaletesting-with-pawsey/index.md

```diff
@@ -0,0 +1,82 @@
+
+During the development of new features, one of the most difficult considerations for Ceph developers is whether a new feature needs to be validated at scale. Our labs allow us to test new features in clusters of up to X,000 OSDs, and in most circumstances this works well!
+
+However, our developers are a cautious bunch and like to test beyond our typical 'test ceiling', if the opportunity arises.
+
+Late last year, such an opportunity presented itself. The **[Pawsey Supercomputing Centre](https://pawsey.org.au/)** based in Perth, Australia reached out with a potential test window. They identified several weeks in their rollout plan where their hardware was ‘racked and stacked’ but would remain unused.
+
+As we were approaching the end of the development cycle for the Quincy release, this was an opportunity too good to miss!
+
+---
+
+## Hardware
+
+Pawsey provided access to over **200** servers, all connected via 100Gb networking. The server specifications were optimized for a large Object storage use case, and enabled us to create the following Ceph cluster;
+
+- 180 OSD Hosts
+- 4,320 OSDs
+- 69 PB (_raw capacity_)
+- 100Gb Networking (_multiple VLANs_)
+- Separate mon/mgr nodes (_'service'_ nodes)
+
+## Test Objectives
+
+Although the orchestrator/cephadm control plane has been in place since the Ceph Octopus release, the challenge of deploying and managing a cluster of this size was defined as our main goal.
+
+In addition to cluster management, we also wanted to assess monitoring/alerting and 'road-test' the experimental `cephadm agent` feature.
+
+## Key Findings
+
+By using Pawsey's cluster we were able to identify and resolve a number of scale issues prior to Quincy's initial release, and identify areas of further work to deliver improvements in the Ceph Reef release.
+
+### Core
+
+- All Ceph daemons report their performance counters to the active manager every 5 seconds by default. We found that the responsiveness of the mgr and CLI became intermittent beyond 2,800 OSDs due to this data load and the interaction with the mgr/prometheus module. A [PR](https://github.com/ceph/ceph/pull/44162) was merged which reduced this issue significantly.
+- A bug was identified in the scale up profile of the [`pg_autoscaler`](https://docs.ceph.com/en/quincy/rados/operations/placement-groups/#autoscaling-placement-groups), which prevented pg’s from peering. This has been subsequently fixed, and backported to [Pacific](https://github.com/ceph/ceph/pull/44869) and [Octopus](https://github.com/ceph/ceph/pull/44541).
+
+### Orchestrator/cephadm
+
+- `ceph-volume` was enhanced to better handle unexpected disk formats like Atari partitions and GPT based devices
+- `cephadm` uses ssh as the control path to each cluster host. During our negative testing, we found that when ssh issues were encountered (bad/missing keys), the resulting information returned to the user was overly verbose. This was fixed in [PR 43880](https://github.com/ceph/ceph/pull/43880).
```

Comment:
not sure if it's relevant for this blog, but that PR didn't actually end up fixing the issue. It later got resolved by https://github.com/ceph/ceph/pull/45132 (and didn't get into the initial Quincy release, which ended up causing issues in the LRC)

Reply:
updated